src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -59,6 +59,54 @@
                    Style="{StaticResource DisclaimerStyle}"
                    Visibility="{x:Bind State.Profile.IsBaseLayer}" />
 
+        <Grid Height="184">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="24" />
+                <RowDefinition Height="160" />
+            </Grid.RowDefinitions>
+            <Grid Grid.Row="0"
+                  Margin="8,0,8,0">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="auto" />
+                </Grid.ColumnDefinitions>

```

Comment:
Feels weird to have a 1-column wide grid. Could you remove the `Grid` and add `Grid.Row=0` and `Margin="8,0,8,0` to the `Border` below?